lib/lib-javadoc/src/main/resources/SQLSpecification2.csv

```diff
@@ -53,6 +53,7 @@ String Functions,Convert all string values from the column foo to upper case.,se
 String Functions,Convert all string values from the column foo to lower case.,select lower(foo) from syn123
 String Functions,Trim leading and trailing white space characters from all string values of the column foo.,select trim(foo) from syn123
 Numeric Functions,Rounds all numeric values in the column foo to 2 decimal places.,"select round(foo,2) from syn123"
+Numeric Functions,"Perform a checksum calculation using the salt, tag, id, and benefactorId to detect if changed have been made to the data.","select sum(crc32(concat('salt','-',etag,'-',id,'-',benefactorId))) from syn123"

```

Comment:
The description should be generic for the CRC32 function. E.g. Perform a checksum calculation of a list of columns.